src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
```

Comment:
[CODE STYLE] Let's use destructuring for the `person` object. It will make your code cleaner and more readable. Refer to the checklist for the correct examples.